src/Core/System/SystemConfig/SystemConfigService.php

```diff
@@ -88,10 +93,16 @@ public function get(string $key, ?string $salesChannelId = null)
                 continue;
             }
 
-            return null;
+            $event = new SystemConfigLoadedEvent($key, null, $salesChannelId);
+            $this->eventDispatcher->dispatch($event);
+
+            return $event->getValue();
         }
 
-        return $pointer;
+        $event = new SystemConfigLoadedEvent($key, $pointer, $salesChannelId);
+        $this->eventDispatcher->dispatch($event);
```

Comment:
not dispatching this even in the config loader, as I actually do not want the transformed value to be stored in the cache (e.g. for secrets it would mean the value would be stored decrypted in the cache which defeats the purpose).